hera_mc/cm_utils.py

```diff
@@ -222,35 +255,6 @@ def get_date_from_pair(d1, d2, ret='earliest'):
         raise ValueError("Must supply earliest/latest.")
 
 
-def get_displayTime(display):
-    if isinstance(display, str):
-        if display.lower() == 'now':
-            d = Time.now()
-        else:
-            d = display
-    elif display is None:
-        d = 'None'
-    elif not isinstance(display, Time):
-        d = 'N/A'
-    else:
-        d = display
-    if isinstance(d, Time):
-        d = "{:%Y-%m-%d %H:%M:%S}".format(d.datetime)
-    return d
-
-
-def get_stopdate(stop_date):
-    if isinstance(stop_date, Time):
-        return stop_date
-    else:
-        return future_date()
-
-
-def is_active(at_date, start_date, stop_date):
-    stop_date = get_stopdate(stop_date)
-    return at_date >= start_date and at_date <= stop_date
-
-
```

Comment:
Note that these functions appear in the diffs even though their code hasn't changed because their position in the file has been reordered. It's better to avoid that when possible to reduce the noise in the diffs.